Assets/MixedRealityToolkit-SDK/Features/Input/Handlers/DragAndDropHandler.cs

```diff
@@ -134,7 +135,7 @@ void IMixedRealityPointerHandler.OnPointerDown(MixedRealityPointerEventData even
             currentPointer = eventData.Pointer;
 
             FocusDetails focusDetails;
-            Vector3 initialDraggingPosition = InputSystem.FocusProvider.TryGetFocusDetails(eventData, out focusDetails)
+            Vector3 initialDraggingPosition = MixedRealityManager.InputSystem.FocusProvider.TryGetFocusDetails(eventData, out focusDetails)
```

Comment:
Not checking for no Input system
```
MixedRealityManager.InputSystem?.
```
Update and check throughout

Reply:
receiving a Pointer event means we have a valid input system.